README.md

```diff
@@ -56,9 +56,44 @@
 
 [![Product Name Screen Shot][product-screenshot]](https://example.com)
 
-Robusta's Kubernetes Resource Recommender (KRR) is a tool that helps users to optimize the resource usage of their Kubernetes clusters. It is based on the Prometheus monitoring system and the Kubernetes API. It is designed to be used both as a CLI tool or as a Kubernetes operator. It is also designed to be easily integrated into Robusta UI, so if you are already using it you can easily start using KRR.
+Robusta KRR (Kubernetes Resource Recommender) is an extensible CLI tool designed to help optimize resource allocation in Kubernetes clusters. It gathers pod usage data from Prometheus, and applies customizable strategies to calculate CPU and memory requests and limits recommendations. This helps you get the most out of your cluster resources and improve overall efficiency.
 
-This tool is also designed to be easily extensible. It is possible to add additional calculation strategies yourself, if you want to use a different strategy than the ones provided by default.
+### Features
+
+-   Extensible Strategies: Easily create and use your own strategies for calculating resource recommendations.
+-   Custom Formatters: Write custom formatters to present the results in your preferred format.
+-   Prometheus Integration: Gather resource usage data using built-in Prometheus queries, with support for custom queries coming soon.
+-   Future Support: Upcoming versions will support custom resources (e.g. GPUs) and custom metrics.
+
+### Resource Allocation Statistics
+
+On average, Kubernetes clusters have:
```

Comment:
Too strong of a statement IMO. I would phrase something like "According to a recent Sysdig study _______"

Keep two points about efficiency, drop the stuff on limits.

Reply:
Also link to the source. Strong claims require proof.